defaults/main.yml

```diff
@@ -13,3 +13,9 @@ RABBITMQ_KEY_FILE: "/etc/letsencrypt/live/{{ RABBITMQ_HOSTNAME }}/privkey.pem"
 RABBITMQ_APT_PACKAGES:
   - python-software-properties
   - python-letsencrypt
+
+RABBITMQ_EXPORT_DIR: /usr/local/lib/rabbitmq/backup
```

Comment:
I changed this to `/var/local/rabbitmq/backup`. Does that seem like a reasonable path?